Eplant/views/InteractionsViewer/InteractionsView.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+import ThumbnailLight from '../../../thumbnails/plant-efp-light.png'
+import { View } from '../../View'
+import PublicationViewerIcon from '../PublicationViewer/icon'
+
+const InteractionsViewer: View = {
+  name: 'Interactions Viewer',
+  id: 'interactions-viewer',
+  component() {
+    return <div>This is not implemented yet!</div>
+  },
+  // TODO: Replace this with correct icon
+  icon: () => <PublicationViewerIcon />,
```

Comment:
This might be confusing to see in DEV. Can you replace with the right icon before merging? Or if we have a default (not implemented) icon that works too.